index.html

```diff
@@ -507,7 +512,66 @@ <h3>StatusList2021Credential</h3>
 <a href="#bitstring-encoding"></a>.
               </td>
             </tr>
-
+            <tr>
+              <td>
+                credentialStatus.ttl
+              </td>
+              <td>
+                The <code>ttl</code> indicates the "time to live" in milliseconds.
+                This property MAY be present, but if not present, implementers MUST 
+                use a value of <code>300000</code> for this property.  A verifier
+                MUST NOT use a cached <code>StatusList2021Credential</code> that was
+                cached more for more than the <code>ttl</code> duration prior to the
+                start of verification operation on a <a>verifiable credential</a>.
+                Implementations that publish the status list SHOULD align
+                any protocol-specific caching information, such as the 
+                HTTP `Cache-Control` header, with the value in this field.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                credentialStatus.size
+              </td>
+              <td>
+                The <code>size</code> indicates the size of the status entry in bits.
+                If size is omitted then the size of each status entry is assumed to be
+                1 bit (and therefore capable of capturing a boolean status).
+                <code>size</code> MAY be provided, but if <code>size</code> is not present
+                as a property of the <code>credentialStatus</code> then <code>size</code>
+                MUST be processed as 1.  <code>size</code> MUST be greater than zero.
+                if <code>size</code> is provided, and is greater than 1, then the property
+                <code>credentialStatus.message</code> MUST be present, and the number of
+                status messages must equal the number of possible values.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                credentialStatus.message
+              </td>
+              <td>
+                The <code>message</code> property MUST be an array, and if present
+                the length of the array must equal the number of possible status states
+                indicated by <code>size</code>. <code>message</code> MAY be present if
+                <code>size</code> is 1. <code>message</code> MUST be present if
+                <code>size</code> is greater than 1.  If not present, the message value
+                associated with the bit value of <code>0</code> is "unset" and the bit
+                value of <code>1</code> is "set".
```

Comment:
Is this "If not present..." optionality valuable? Why not require `message` (or `enumeration` or whatever if we rename it) if the `statusPurpose` says that it's being used?

Reply:
At this time, to permit use with additional status purposes that are not yet defined, but that may want multiple status codes.

Reply:
> At this time, to permit use with additional status purposes that are not yet defined, but that may want multiple status codes.

Hmm, this says that `message` is only optional if the size is 1. That would eliminate the possibility of multiple status codes (it just tracks one thing) for that list, wouldn't it?

If the status list was being defined with `size > 1`, I can see how perhaps not every status value would have semantics defined at the outset, but `message` is required in that case. The text also says that if `message` is present, then `value` must also be. So I'm not sure this optionality is needed for the case of `size = 1`, why not require it just the same? I think it's safe to assume that any list that is being created has, minimally, one purpose, else it wouldn't be (or shouldn't be) created. And for lists of `size = 1`, it can only track one thing anyway.

In short, I'd think that if a status list were being defined with `size = 1`, then what that list is tracking should be known / defined, otherwise don't create the list.

Reply:
I think we can expand this out with examples in #66 and refine as we go

Reply:
this also sets a default status message if you set `statusPurpose` as `status` and a size of 1

Reply:
Can you change size from 1 to a number greater than 1 after the initial status list has been created?